_episodes/03-github-pages.md

```diff
@@ -3,275 +3,126 @@ title: "Hosting Pages and Websites on GitHub"
+  (Markdown, HTML, CSS, JavaScript, etc.)
+  straight from a repository on GitHub, runs them through its website engine Jekyll, builds them into a website,
+  and publishes them on the Web"
+- "By convention, if you create a branch called `gh-pages` in your repository, it will automatically be published as a website by GitHub"
+- "You can configure any branch of a repository to be used for website (it does not have to be `gh-pages`)"
+- "GitHub publishes websites on special URLs formatted as 'https://GITHUB_USERNAME.github.io/REPOSITORY_NAME'"
+
 ---
 
-# What is GitHub Pages?
+Now that you know how to create Markdown files, let's see how to turn them into web pages. GitHub has a 
+service just for that called GitHub Pages.  
+
+# Publishing a Repository as a Website With GitHub Pages
 [GitHub Pages](https://docs.github.com/en/github/working-with-github-pages/about-github-pages) is a free website
-hosting service by GitHub that takes files from your GitHub repository that is configured as a website,
-optionally runs the files through a build process, and publishes them as a website or a web page.
+hosting service by GitHub that takes files (Markdown, HTML, CSS, JavaScript, etc.) 
+from your GitHub repository which is configured as a website,
+optionally runs the files through a build process, combines them and publishes them as a website or a Web page.
 Any changes you do to the files in your website's GitHub repository
 will be rendered live in the website.
 
-# Publishing a repository as a website
-
-GitHub Pages is a service from GitHub to publish a website directly from a Git repository.
-
-> ## Why Github?
->
-> There are other services available to create and publish website but one of the main advantage of GitHub Pages is that you can version control your website and therefore keep track of all your changes. This is particularly helpful for collaborating on a project website. [GitLab](https://about.gitlab.com/) offers very similar services but GitHub pages is the simplest approach.
->
-{: .callout}
-
-Let's start from the github repository we have created in the previous episode {{ site.project-pages }}. One important file you should have is `index.md`: it will become the homepage of your project website.
-
-In this episode, you will learn how to turn your project repository into a website.
+There are other services available to create and publish website but one of the main advantages of GitHub Pages is that you can version control your website and therefore keep track of all your changes. This is particularly helpful for collaborating on a project website. [GitLab](https://about.gitlab.com/) offers very similar services but GitHub pages is the simplest approach.
 
-## Turn on GitHub Pages for your main branch
+Let's continue from the GitHub repository we have created in the previous episode. One important file you should have already is `index.md`: it will become the homepage of your project website.
```

Comment:
I think the learners will only have the `README.md` file at the starting point of this lesson. In the absence of a file called `index.md`, the README is used as the landing page of the GitHub Pages site, so it shouldn't change anything for this section, except adding a part to create a separate `index.md` later...

Reply:
Oh, I've missed that bit of discussion and have just added exercises that work with `index.md`. It's not a biggie, I will  add a bit where we create it.

Reply:
Fixed via 596a28b.